app/src/main/java/org/mifos/selfserviceapp/utils/Toaster.java

```diff
@@ -38,6 +38,13 @@ public static void show(View view, int res, int duration) {
         show(view, MifosSelfServiceApp.getContext().getResources().getString(res), duration);
     }
 
+    public static void cancelTransfer(View view, String text, String buttonText,
```

Comment:
@luckyman000786, for now, seems good, I will think about better implementation later but this is good for now. thanks